rubicon-htb.js

```diff
@@ -254,6 +256,35 @@ function RubiconModule(configs) {
         return firstPartyData;
     }
 
+    function _getDigiTrustQueryParams() {
+        if (configs.digitrustId) {
+            _digiTrustId = configs.digitrustId;
+        }
+            function getDigiTrustId() {
+                if (!Browser.isTopFrame()) {
+                    try {
+                    var _window = window.top;
+                    } catch(e) {
+                        console.log("impossible to reach top window");
+                    }
+                } else {
+                    var _window = window;
+                }
+                var digiTrustUser =  (_digiTrustId || _window.DigiTrust.getUser({member: 'T9QSFKPDN9'})) 
```

Comment:
Please add additional try-catch as Window my not have DigiTrust and this can cause the wrapper to throw errors.

Reply:
Thanks will add the try catch